Realm/RLMSchema.mm

```diff
@@ -126,12 +126,20 @@ - (instancetype)init {
 
 - (NSArray *)objectSchema {
     if (!_objectSchema) {
+        if (self == s_sharedSchema) {
+            // Ensure the schema is fully initialized
+            [RLMSchema sharedSchema];
```

Comment:
The schema is only fully initialized when the array `objectSchema` is accessed. If a scheme is looked up by name, it doesn't load the entire schema.